vc-verifier/js/src/constant/CredentialVerifierConstants.js

```diff
@@ -0,0 +1,6 @@
+export const CredentialVerifierConstants = {
```

Comment:
It should be Constants or CredentialVerifierConstants?

Reply:
Done.Changed it.